go.mod

```diff
@@ -100,3 +99,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.7.0 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace github.com/google/gopacket v1.1.19 => github.com/GameFabric/sts-pkg-gopacket v0.0.0-20230307093013-7b513277b714
```

Comment:
This points on an open PR which never been merged: https://github.com/google/gopacket/pull/1076

Let me know if you prefer to have our own implementation in netobserv org

Reply:
is there a reason to use google/gopacket rather than gopacket/gopacket? I kind of remember in the ebpf agent we switch to the latter, can't remember the exact reason, but I think gopacket/gopacket was more up to date.
And it sounds like it also have ngread: https://github.com/search?q=repo%3Agopacket%2Fgopacket%20ngread&type=code

Reply:
I compared `GameFabric/sts-pkg-gopacket` to  `gopacket/gopacket` and it doesn't contains the comments capabilities I'm looking for:

![image](https://github.com/user-attachments/assets/4c35a0bb-bd2c-4102-97a3-15e10e207b8c)

So I need to keep the `replace` part wich forces me to keep the google one since the modules been renamed between the two:
```
module declares its path as: github.com/google/gopacket
	        but was required as: github.com/gopacket/gopacket
```

Unless we want to have our own implementation in netobserv repos as mentionned before ?

Reply:
So I think we need to fork it. The replacement used here seems to be from 2023, could be pretty bad for security...